contentcuration/contentcuration/utils/celery_signals.py

```diff
@@ -17,6 +21,24 @@
 logger = get_task_logger(__name__)
 
 
+def clear_and_connect():
+    try:
+        connection.cursor()
+    except InterfaceError:
+        connection.close_if_unusable_or_obsolete()
+        connection.connect()
+
+
+@task_prerun.connect
+def prerun(sender, **kwargs):
+    clear_and_connect()
+
+
+@task_postrun.connect
+def postrun(sender, **kwargs):
+    clear_and_connect()
```

Comment:
If we're clearing on every new task, is clearing them after every task redundant?